google/oauth2/credentials.py

```diff
@@ -90,6 +91,17 @@ def client_secret(self):
         return self._client_secret
 
     @property
+    def refresh_grant_response(self):
```

Comment:
I was on team "no new public properties", so neither really fits that.

Though given your description, it's a bit strange to tell people to access the things they want in two slightly different ways:

```python
>>> creds.access_token
>>> creds.refresh_grant_response['id_token']
```

so if you were going to add anything, make it the ID token?